demos/browser/script/conditional-install

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
```

Comment:
Why cant we reuse the one in the main package?

Reply:
It's a short script and paths get confusing.